go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/Masterminds/semver/v3 v3.1.1
 	github.com/dyson/certman v0.2.1
 	github.com/giantswarm/apiextensions-application v0.3.1
-	github.com/giantswarm/app/v6 v6.8.1
+	github.com/giantswarm/app/v6 v6.9.1-0.20220519155702-427ee1cdf0c3
```

Comment:
This will need an update when https://github.com/giantswarm/app/pull/242 gets merged, right?

Reply:
Yup, this commit version is here just for testing